src/main.py

```diff
@@ -122,7 +119,6 @@ def donation_prompt(ty_answer, mode='prompt', mode2='prompt'):
     else:
         DONORS[ty_answer].update({donation_date: int(donation_amt)})
         return DONORS[ty_answer][donation_date]
-    return ty_answer
 
 
 # def donation_date(donation_amt, ty_answer, mode='prompt'):
```

Comment:
Make sure you always purge any commented-out code aka dead code from your final submission.